extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,269 @@
+  background: var(--i-amphtml-story-dev-tools-device-chip-background) !important;
+  border: 1px solid var(--i-amphtml-story-dev-tools-device-chip-background) !important;
+  height: 32px !important;
+  font-size: 13px !important;
+  display: inline-flex !important;
+  align-items: center !important;
+  padding-inline-start: 10px !important;
+  padding-inline-end: 6px !important;
+  margin: 0 5px !important;
+  white-space: nowrap !important;
+  cursor: pointer !important;
+  outline: none;
+}
+
+.i-amphtml-story-dev-tools-device-chip:hover, .i-amphtml-story-dev-tools-device-chip:focus {
+  background: var(--i-amphtml-story-dev-tools-device-chip-background-hover) !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip svg {
+  cursor: pointer !important;
+  background: var(--i-amphtml-story-dev-tools-device-chip-icon-background) !important;
+  border-radius: 50% !important;
+  padding: 2px !important;
+  margin-inline-start: 10px !important;
+  transition: transform 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] {
+  --i-amphtml-story-dev-tools-device-chip-background: transparent !important;
+  --i-amphtml-story-dev-tools-device-chip-background-hover: var(--i-amphtml-story-dev-tools-device-gray-1) !important;
+  --i-amphtml-story-dev-tools-device-chip-icon-background: var(--i-amphtml-story-dev-tools-device-white-transparent-1) !important;
+  border: 1px solid var(--i-amphtml-story-dev-tools-device-chip-icon-background) !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip[inactive] svg {
+  transform: rotate(90deg) !important;
+}
+
+/* Removes the second line in the + (to convert to -) when the device is selected (not inactive) */
+.i-amphtml-story-dev-tools-device-chip:not([inactive]):not(.i-amphtml-story-dev-tools-add-device) .i-amphtml-story-dev-tools-device-chip-add-stick {
+  fill: transparent !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device {
+  background: var(--i-amphtml-story-dev-tools-accent) !important;
+  color: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device svg {
+  fill: var(--i-amphtml-story-dev-tools-accent) !important;
+  background: black !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:hover, .i-amphtml-story-dev-tools-device-chip.i-amphtml-story-dev-tools-add-device:focus {
+  background: var(--i-amphtml-story-dev-tools-accent-light) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg {
+  position: fixed !important;
+  left: 0 !important;
+  top: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: transparent !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  z-index: 1 !important;
+  transition: background 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
```

Comment:
This should use the standard curve

Reply:
This is standard curve, appended `-standard` to the name to make it clear